examples/layersColorVisible.html

```diff
@@ -0,0 +1,63 @@
+<!-- this example outputs in console, the array of colors layers really displayed in viewer -->
```

Comment:
Sorry but I still don't think it's worth adding this to the examples.
But I agree the content is useful: it should be in the documentation.